addons/core/functions/flexiUI/flexiInit.sqf

```diff
@@ -1,2 +1,2 @@
-["TFAR","OpenSWRadioMenu",["Open SW Radio Menu","Open SW Radio Menu"],{["player",[],-3,"_this call TFAR_fnc_swRadioMenu",true] call cba_fnc_fleximenu_openMenuByDef;},{true},[TF_dialog_sw_scancode, TF_dialog_sw_modifiers],false] call cba_fnc_addKeybind;
-["TFAR","OpenLRRadioMenu",["Open LR Radio Menu","Open LR Radio Menu"],{["player",[],-3,"_this call TFAR_fnc_lrRadioMenu",true] call cba_fnc_fleximenu_openMenuByDef;},{true},[TF_dialog_lr_scancode, TF_dialog_lr_modifiers],false] call cba_fnc_addKeybind;
+["TFAR","OpenSWRadioMenu",["STR_TFAR_openSWMenu","STR_TFAR_openSWMenu"],{["player",[],-3,"_this call TFAR_fnc_swRadioMenu",true] call cba_fnc_fleximenu_openMenuByDef;},{true},[TF_dialog_sw_scancode, TF_dialog_sw_modifiers],false] call cba_fnc_addKeybind;
+["TFAR","OpenLRRadioMenu",["STR_TFAR_openLRMenu","STR_TFAR_openLRMenu"],{["player",[],-3,"_this call TFAR_fnc_lrRadioMenu",true] call cba_fnc_fleximenu_openMenuByDef;},{true},[TF_dialog_lr_scancode, TF_dialog_lr_modifiers],false] call cba_fnc_addKeybind;
```

Comment:
@dedmen 
I reference some file, seen like when call CBA_A3 function,  stringtable don't havve to use localize or $ symbol, is it right ?

Reply:
I think in this case CBA does it automatically internally.